package.json

```diff
@@ -45,11 +54,11 @@
     "split": "^1.0.0"
   },
   "scripts": {
-    "test": "./bin/run-test.sh",
-    "system-test": "./bin/run-system-test.sh",
-    "changelog": "./bin/run-changelog.sh",
-    "coverage": "./bin/run-test.sh -c",
-    "bump": "./bin/run-bump.sh",
-    "closure": "./node_modules/.bin/closure-npc"
+    "test": "gulp test",
```

Comment:
It seems that local testing by developers will be different from what travis runs. Intentional? if so, why?